src/pageObjects/WelcomePage/components/SignUpPopup.js

```diff
@@ -0,0 +1,39 @@
+import BaseComponent from '../../../components/BaseCoomponent';
+import GaragePage from '../../GaragePage/GaragePage';
+
+export default class SignUpPopup extends BaseComponent {
+  _nameInputSelector = '#signupName';
+
+  _lastNameInputSelector = '#signupLastName';
+
+  _emailInputSelector = '#signupEmail';
+
+  _passwordInputSelector = '#signupPassword';
+
+  _repeatPasswordInputSelector = '#signupRepeatPassword';
+
+  _invalidFieldSelector = 'div.invalid-feedback';
+
+  _registerBtnSelector = 'button:has-text("Register")';
+
+  constructor(page) {
+    super(page, page.locator('app-signup-modal'));
+    this.nameField = this.container.locator(this._nameInputSelector);
+    this.lastNameField = this.container.locator(this._lastNameInputSelector);
+    this.emailField = this.container.locator(this._emailInputSelector);
+    this.passwordField = this.container.locator(this._passwordInputSelector);
+    this.repeatPasswordField = this.container.locator(this._repeatPasswordInputSelector);
+    this.invalidField = this.container.locator(this._invalidFieldSelector);
```

Comment:
You should have an ability to check an error for each input

with this selector you do not know where error appears

Reply:
Selector is fixed so that it checks specific field